template/cookiecutter.json

```diff
@@ -1,7 +1,8 @@
 {
     "project_name": "my-project",
-    "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
     "repo_host": ["github", "hgmo"],
+    "repo_name": "my-repo",
```

Comment:
what's the difference between repo_name and project_name?

Reply:
Repo name is the name of the Github repository, project name is the name of the local checkout dir. Usually they're the same thing, but this fix is a result of that not being the case.